src/TeamSpeak3QueryApi/Enums/Protocol.cs

```diff
@@ -0,0 +1,12 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace TeamSpeak3QueryApi.Net.Enums
+{
+    public enum Protocol
```

Comment:
What is the enum used for?

```
QueryClient c0 = new SshQueryClient();
QueryClient c1 = new RawQueryClient();
```
...would not need such an enum.

Reply:
The interface was needed cause the query client hasn´t the methods. Now was a abstract declaration we don´t need this interface anymore. Thanks for this information 👍